mindjunkies/courses/views.py

```
@@ -9,9 +9,10 @@
 from django.views.decorators.http import require_http_methods
 from django.views.generic import TemplateView
 from django.views.generic.edit import CreateView, UpdateView
+from django.views.generic import TemplateView
```

Ruff (0.8.2)

Redefinition of unused `TemplateView` from line 10

Remove definition: `TemplateView`

(F811)